csharp/src/Ice/TcpAcceptor.cs

```diff
@@ -90,11 +90,11 @@ public virtual ITransceiver Accept()
             Socket acceptFd = _acceptFd;
             _acceptFd = null;
             _acceptError = null;
-            return _endpoint.CreateTransceiver(Transport,
+            return _endpoint.CreateTransceiver(TransportName,
                 new StreamSocket(_communicator, acceptFd), _adapterName);
         }
 
-        public string Transport { get; }
+        public string TransportName => _endpoint.TransportName;
```

Comment:
Can't we just get rid of this `TransportName` property and of this `TransportName` parameter of `CreateTransceiver`?

Reply:
Maybe but it's not trivial to remove. For a future PR.